packages/node/src/executeBatch.ts

```diff
@@ -50,24 +48,19 @@ const DEFAULT_STATE: StateMapValue = {
 function canExecuteTransaction(state: StateMap, tx: Transaction): boolean {
   try {
     // Step 0. Check transaction type
-    if (!(tx.from !== Hex(0) && tx.to !== Hex(0))) {
```

Comment:
@mateuszradomski was this a bug?

Reply:
Yes, most likely I meant to compare to EthereumAddress.ZERO instead of Hex(0)